### PR TITLE
[JENKINS-39454] Do not consider pendings from scheduleInternal

### PR DESCRIPTION
**Pull request by jglick**
Hard to explain, so read [JENKINS-39454](https://issues.jenkins-ci.org/browse/JENKINS-39454).

@reviewbybees

### CODE REVIEW

File: core/src/main/java/hudson/model/Queue.java [hunk 1]
```diff
@@ -1414,7 +1423,7 @@ public void maintain() {
                             p.task.getFullDisplayName());
                     p.isPending = false;
                     pendings.remove(p);
-                    makeBuildable(p);
+                    makeBuildable(p); // TODO whatever this is for, the return value is being ignored, so this does nothing at all
```
Comment (jglick) [inline core/src/main/java/hudson/model/Queue.java]: Looks like an unrelated bug. See #2510; we occasionally get here, but I have never figured out what this case means, and apparently this line does absolutely nothing. Maybe a regression from #1513? Clearly not covered by any existing tests.

**Review by daniel-beck — APPROVED** [review]

**Comment by oleg-nenashev** [comment]
Needs some testing in the field, but 🐝after reading the ticket and googling for the code.
@reviewbybees done

File: core/src/main/java/hudson/model/Queue.java (hunk 1 above)
Comment (jglick) [reply core/src/main/java/hudson/model/Queue.java:1426]: ⇒ https://github.com/jenkinsci/jenkins/pull/10756
